src/x86/sse.rs

```diff
@@ -165,6 +166,188 @@ pub unsafe fn _mm_max_ps(a: f32x4, b: f32x4) -> f32x4 {
     maxps(a, b)
 }
 
+/// Bitwise AND of packed single-precision (32-bit) floating-point elements.
+#[inline(always)]
+#[target_feature = "+sse"]
+// i586 only seems to generate plain `and` instructions, so ignore it.
```

Comment:
This seems a little worrisome, I wonder if we don't have the right codegen for that then?

Reply:
Er well [apparently this is normal](https://github.com/llvm-mirror/llvm/blob/7e5d086d1e5ede0d73f51fca55dd1fdd6adfc796/test/CodeGen/X86/sse-intrinsics-fast-isel.ll#L38-L70)!